frontend/src/stores/authStore.js

```diff
@@ -6,12 +6,21 @@ const authStore = {
     username: '',
     email: '',
     roles: [],
+    clientId: '',
+    clientSecret: '',
   },
   saveState(user) {
+    console.log(user);
```

Comment:
The console logs can get removed from the auth store.

Reply:
Should be removed.

Reply:
Code has been updated.